pallets/tokens/src/multi_token_currency.rs

```diff
@@ -345,6 +348,7 @@ pub trait MultiTokenCurrencyExtended<AccountId>: MultiTokenCurrency<AccountId> {
         address: &AccountId,
         amount: Self::Balance,
     ) -> DispatchResult;
+    fn get_next_currency_id() -> Self::CurrencyId;
```

Comment:
doesnt really seem to be `CurrencyAttribute` isnt there a better to propagate that information?

Reply:
after rethinking it really seems to be ok ...